File: KITTY/KURS_1/JAVA/DAY_25_Set_Preference/app/src/main/java/com/example/velic/day_25_set_preference/MyPreferenceScreen.java
package com.example.velic.day_25_set_preference;

import android.os.Bundle;
import android.preference.PreferenceActivity;

public class MyPreferenceScreen extends PreferenceActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.preferencescreen);
    }

}

File: KITTY/KURS_1/KOTLIN/DAY_25_1_Set_Preference_kt/app/src/main/java/com/example/day_25_1_set_preference_k/MyPreferenceScreen.kt
package com.example.day_25_1_set_preference_k

import android.os.Bundle
import android.preference.PreferenceActivity

class MyPreferenceScreen : PreferenceActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.preferencescreen)
    }

}
